README.md

```diff
@@ -1,29 +1,28 @@
 llvm-cbe
 ========
 
-resurrected LLVM "C Backend", with improvements
+Resurrected LLVM C backend, with improvements.
 
-
-INSTALLATION INSTRUCTIONS
+Installation instructions
 =========================
 
-This version of the LLVM-CBE library works with LLVM 8.0. You will have
-to compile this version of LLVM before you try to use LLVM-CBE. This
-guide will walk you through the compilation and installation of both
-tools and show usage statements to verify that the LLVM-CBE library is
-compiled correctly.
+This version of the LLVM C backend works with LLVM 10.0.
 
-The library is known to compile on various Linux versions (Redhat,
-Mageia, Ubuntu, Debian), Mac OS X, and Windows (Mingw-w64).
+Earlier versions are supported too (LLVM 7 and earlier should work).
 
 Step 1: Installing LLVM
 =======================
 
-LLVM-CBE relies on specific LLVM internals, and so it is best to use
-it with a specific revision of the LLVM development tree. Currently,
-llvm-cbe works with the LLVM 8.0 release version and autotools.
+Either install the LLVM packages on your system:
+--------------------------------------------
+
+On macOS, use pkgsrc and install the llvm package. On CentOS, install the llvm-devel package. On Debian and derivatives, install the llvm-dev package.
+
+It is recommended to install the clang package too.
```

Comment:
```suggestion

Install LLVM and Clang:

- Ubuntu, Fedora, (Debian and derivatives)
```shell
sudo apt-get install wget
wget https://apt.llvm.org/llvm.sh
chmod +x llvm.sh
sudo ./llvm.sh 10
```

- Windows:
install [choco](https://chocolatey.org/install) then:
```shell
choco install llvm
```

- MacOS:
install [pkgsrc](https://www.pkgsrc.org/) then
```shell
# ??
```

- CentOS
```
yum install llvm-devel
yum install clang-devel
```

```

Reply:
@woachk please add the macOS command. I don't know it.

Reply:
Hello, 

I wouldn't recommend using the LLVM.org build from apt.llvm.org because staying with the system LLVM shipped with the OS..

For Windows, no need for Chocolatey, you can take the binary installer from llvm.org. ;)

For Mac, instructions to install pkgsrc are on http://pkgsrc.joyent.com/install-on-osx/, and then the command is pkgin in llvm clang.

Reply:
> I wouldn't recommend using the LLVM.org build from apt.llvm.org because staying with the system LLVM shipped with the OS..

@woachk I have used it and works perfectly fine. No OS now has LLVM-10 installed on by default. That script automatically install proper package based on the distro.

> For Windows, no need for Chocolatey, you can take the binary installer from llvm.org. ;)
> For Mac, instructions to install pkgsrc are on http://pkgsrc.joyent.com/install-on-osx, and then the command is pkgin in llvm clang.

Thanks! I Update the suggestion.

Reply:
About almost no OSes having LLVM 10 on purpose, yeah that's why I'm for supporting older LLVM versions too, and keep the capability to do so. ;)

Not wanting to use packages with another LLVM version because mixing between versions can end up being quite nasty. It especially matters nowadays when GPU drivers use LLVM on Linux today as a part of their shader compilation pipeline.